hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java

```diff
@@ -1912,9 +1908,8 @@ public ReplicaHandler createTemporary(StorageType storageType,
       return new ReplicaHandler(newReplicaInfo, ref);
     } finally {
       if (dataNodeMetrics != null) {
-        // Create temporary operation hold write lock twice.
-        long createTemporaryOpMs = Time.monotonicNow() - startHoldLockTimeMs
-            + holdLockTimeMs;
+        // Create temporary operation hold write lock once.
+        long createTemporaryOpMs = Time.monotonicNow() - startHoldLockTimeMs;
```

Comment:
I think `createTemporaryOpMs` should be `Time.monotonicNow() - startTimeMs`;

Reply:
@tomscut thanks for the help review.

Update PR, Please help review again.